client/src/components/v2/CameraRecording.js

```diff
@@ -1,13 +1,13 @@
-import React from 'react';
-import { Button, Card, Col, Row } from 'react-bootstrap';
-import PropTypes from 'prop-types';
 import {
-  startRecording,
+  getPrettyDeviceName, startRecording,
```

Comment:
This could be on two lines. Weird that the linter didn't complain about this!

Reply:
Not sure why either. I seem to be missing a few of the eslint rules that we have; perhaps some of the rules are being used from a global `.eslintrc` rather than the local one?

![image](https://user-images.githubusercontent.com/50545432/90727796-331c7280-e307-11ea-98b6-8244bca24662.png)

Done
9ecf459